docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-powershell-keylogging-script.asciidoc

```diff
@@ -0,0 +1,149 @@
+[[prebuilt-rule-0-16-2-powershell-keylogging-script]]
+=== PowerShell Keylogging Script
+
+Detects the use of Win32 API Functions that can be used to capture user keystrokes in PowerShell scripts. Attackers use this technique to capture user input, looking for credentials and/or other valuable data.
```

Comment:
```suggestion
Detects the use of Win32 API Functions that can be used to capture user keystrokes in PowerShell scripts. Attackers use this technique to capture user input and look for credentials and/or other valuable data.
```

Reply:
Flagging this in case the second sentence was supposed to be something else, like:

Attackers use this technique to capture user input that's looking for credentials and/or other valuable data.